### ISSUE TITLE
conversion not working in windows 10

### ISSUE THREAD

**Issue by ddontham** [snoopyjc/xls2xlsx#2]
* xls2xlsx version:
* Python version: 3.8.2
* Operating System: Windows 10

### Description
i created a simple xls file and tried to use the utility from command line it is give the below error

![image](https://user-images.githubusercontent.com/68939440/96094628-35581300-0eeb-11eb-8fda-da11b4c1d0cc.png)

```


![image](https://user-images.githubusercontent.com/68939440/96094846-7cde9f00-0eeb-11eb-8c93-9a86784d8527.png)

**Comment by ddontham** [comment]
![image](https://user-images.githubusercontent.com/68939440/96094884-87009d80-0eeb-11eb-8da4-c40453ef4a0d.png)

**Comment by snoopyjc** [comment]
Sorry about that - looks like I have no tests for the cli (command line interface).  Anyhow I know what the problem is and will push a fix tonight.

**Comment by snoopyjc** [comment]
Fixed in v0.1.3
